opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -40,6 +40,18 @@
 #include <opm/parser/eclipse/Deck/Deck.hpp>
 #include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
 
+
+namespace {
+    inline
+    void silenceCompilerWarnings() {
```

Comment:
I have seen warnings for these, but strangely only when there are compiler errors. Are you seeing warnings for these even for a successful compile?

Also: why is it placed in this file? It should probably be placed in opm-core's Units.hpp if it is required.